.github/workflows/release-macos.yml

```diff
@@ -162,19 +162,13 @@ jobs:
         rm -f bundles/*.sha256
         echo "::endgroup::"
 
-    - name: Install gon
-      env:
-        HOMEBREW_NO_AUTO_UPDATE: 1
-        HOMEBREW_NO_INSTALL_CLEANUP: 1
-      run: |
-        brew tap mitchellh/gon
-        brew install mitchellh/gon/gon
-
     - name: Notarize
       env:
         AC_USERNAME: ${{ secrets.APPLE_DEVELOPER_APP_USERNAME }}
         AC_PASSWORD: ${{ secrets.APPLE_DEVELOPER_APP_PASSWORD }}
+        AC_TEAM_ID: ${{ secrets.APPLE_DEVELOPER_TEAM_ID }}
       run: |
+        xcrun notarytool store-credentials --apple-id "$AC_USERNAME" --password "$AC_PASSWORD" --team-id "$AC_TEAM_ID" openttd
```

Comment:
This will now fail for people who don't have those secrets, read: everyone but us. Shouldn't we silent ignore errors, like we did before?

Sadly, an `if:` in the workflow itself doesn't work, as that cannot check if a secret exists .. so in bash an `if` statement seems the way to go?

Reply:
The workflow fails anyway if you don’t have a survey secret key set up, or something, I discovered. 🙂

I can add a bash if check though.